target/appengine-staging/__static__/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
```

Comment:
You shouldn't have any changes to this file. SO I would recommend not includding it in the PR.